odr-audioenc.spec

```diff
@@ -51,7 +51,7 @@
 
 Name:           %{main_name}
 Version:        %{main_version}
-Release:        %{main_release}
+Release:        %{main_release}.1
```

Comment:
Can we add this to the `%define main_release`?

Reply:
Yeah sure, `rpmdev-bumpspec` broke them and I didn't notice. Thanks for spotting it.